application/backend/src/main/resources/apps/typerefinery/components/widgets/ticker/script.js

```diff
@@ -100,7 +101,13 @@ $(document).ready(function (e) {
 
     // Data can be updated via TMS Connection.
     if (componentTopic && componentHost) {
-      component.setAttribute("id", componentTopic);
+      // component.setAttribute("id", componentTopic);
```

Comment:
Why is this line commented on?
Is there any reason?

If you comment on this then Line 36 will not work.

Reply:
Resolved